packages/flutterfire_cli/lib/src/commands/config.dart

```diff
@@ -117,13 +117,49 @@ class ConfigCommand extends FlutterFireCommand {
           'and create the google-services.json file in your ./android/app folder.',
     );
     argParser.addFlag(
-      'debug-symbols-script',
-      hide: true,
-      abbr: 'd',
+      'debug-symbols-ios',
+      abbr: 'r',
```

Comment:
```suggestion
      'debug-symbols-ios',
      abbr: 'dsi',
```

Reply:
Unfortunately, we cannot change abbreviations to anything more than single characters 😓, @Lyokone

Reply:
Oh that's a shame :/